io.github.zer0x.qurancompanion.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.zer0x.qurancompanion
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: quran-companion
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=session-bus
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --device=dri
+modules:
+  - name: extras
```

Comment:
this should be part of the "main" modules. Also all these file should be submitted upstream.